dependencies.props

```diff
@@ -195,4 +199,8 @@
     <!-- project.json files to update -->
     <ProjectJsonFiles Include="$(MSBuildThisFileDirectory)external\**\optional.json" />
   </ItemGroup>
+
+  <!-- Override isolated build dependency versions with versions from Repo API. -->
+  <Import Project="$(DotNetPackageVersionPropsPath)"
```

Comment:
Is there any reason to not include this in the repoapi targets in BuildTools?

Reply:
The order matters here: first, Build.Common.props get imported:

https://github.com/dotnet/corefx/blob/ed0ee133ac49cee86f10ca4692b1d72e337bc012/dir.props#L51

Then dependencies.props:

https://github.com/dotnet/corefx/blob/ed0ee133ac49cee86f10ca4692b1d72e337bc012/dir.props#L66

In theory we could change it up:
 1. Import the package version props in BuildTools
 1. In CoreFX (dependencies.props and all project files), use nonstandard names for the package version properties to avoid overwriting what BuildTools brought in
 1. Add a PropertyGroup here instead of the import that maps every package version prop onto the corresponding CoreFX-specific name that the projects used.

I think it's better to have the import here to avoid that mapping table.

Reply:
Sounds good